File: python/12_file_io.py
# 12 file i/o

with open("test.txt", "w") as f:
    f.write("hello from python\n")
    f.write("second line\n")

with open("test.txt", "r") as f:
    content = f.read()
    print(content)

with open("test.txt", "r") as f:
    for line in f:
        print("line:", line.strip())

File: rust/12_file_io.rs
use std::fs;

fn main() {
    fs::write("test.txt", "hello from rust\nsecond line\n").expect("write failed");

    let content = fs::read_to_string("test.txt").expect("read failed");
    println!("{}", content);

    for line in content.lines() {
        println!("line: {}", line);
    }
}
